ui/app/templates/components/link-status.hbs

```diff
@@ -8,8 +8,59 @@
           HCP.
         </a>
       {{else}}
-        {{this.message}}
+        There was an error connecting to HCP. Click
+        <button type="button" class="text-button is-underline" {{on "click" (fn (mut this.showModal))}}>
+          here
+        </button>
+        for more information.
       {{/if}}
     </p>
   </div>
-{{/if}}
\ No newline at end of file
+{{/if}}
+
+{{#if this.isTesting}}
+  <div id="modal-wormhole"></div>
+{{/if}}
```

Comment:
This prevents having to add the wormhole to every component test that uses `nav-header`